*** libavcodec/avcodec.h.old	Sat Jun 27 18:45:05 2009
--- libavcodec/avcodec.h	Sat Jun 27 18:45:04 2009
***************
*** 568,573 ****
--- 568,575 ----
  #define CODEC_FLAG2_CHUNKS        0x00008000 ///< Input bitstream might be truncated at a packet boundaries instead of only at frame boundaries.
  #define CODEC_FLAG2_NON_LINEAR_QUANT 0x00010000 ///< Use MPEG-2 nonlinear quantizer.
  #define CODEC_FLAG2_BIT_RESERVOIR 0x00020000 ///< Use a bit reservoir when encoding if possible
+ //MEANX: NEVER EVER USE CLOSED GOP ?
+ #define CODEC_FLAG2_32_PULLDOWN   0x80000000 
  
  /* Unsupported options :
   *              Syntax Arithmetic coding (SAC)
***************
*** 1422,1427 ****
--- 1424,1430 ----
       * - decoding: unused
       */
      int rc_max_rate;
+     int rc_max_rate_header; /*< That one is set in the header MEANX */
  
      /**
       * minimum bitrate
***************
*** 1436,1441 ****
--- 1439,1446 ----
       * - decoding: unused
       */
      int rc_buffer_size;
+     int rc_buffer_size_header;  /*< That one is set in the header MEANX*/
+ 
      float rc_buffer_aggressivity;
  
      /**
