--- libavcodec/avcodec.h.orig	2011-05-10 22:40:28 +0100
+++ libavcodec/avcodec.h	2011-05-10 22:40:28 +0100
@@ -636,6 +636,8 @@
 #define CODEC_FLAG2_PSY           0x00080000 ///< Use psycho visual optimizations.
 #define CODEC_FLAG2_SSIM          0x00100000 ///< Compute SSIM during encoding, error[] values are undefined.
 #define CODEC_FLAG2_INTRA_REFRESH 0x00200000 ///< Use periodic insertion of intra blocks instead of keyframes.
+//MEANX: NEVER EVER USE CLOSED GOP ?
+#define CODEC_FLAG2_32_PULLDOWN   0x80000000
 
 /* Unsupported options :
  *              Syntax Arithmetic coding (SAC)
@@ -1589,6 +1591,7 @@
      * - decoding: unused
      */
     int rc_max_rate;
+    int rc_max_rate_header; /*< That one is set in the header MEANX */
 
     /**
      * minimum bitrate
@@ -1603,6 +1606,8 @@
      * - decoding: unused
      */
     int rc_buffer_size;
+    int rc_buffer_size_header;  /*< That one is set in the header MEANX*/
+
     float rc_buffer_aggressivity;
 
     /**
