--- ../ffmpeg-0.8/libavcodec/mpeg12.c	2011-06-23 20:23:54 +0100
+++ libavcodec/mpeg12.c	2011-06-23 20:23:54 +0100
@@ -1998,6 +1998,11 @@
         ff_er_frame_end(s);
 
         MPV_frame_end(s);
+/* MEANX */
+  if(s->current_picture_ptr)
+      s->current_picture_ptr->opaque=pict->opaque;
+/* MEANX */
+
 
         if (s->pict_type == AV_PICTURE_TYPE_B || s->low_delay) {
             *pict= *(AVFrame*)s->current_picture_ptr;
