*** libavcodec/avcodec.h.old	Sat Jan 24 22:23:50 2009
--- libavcodec/avcodec.h	Sat Jan 24 22:23:50 2009
***************
*** 465,470 ****
--- 465,472 ----
  #define CODEC_FLAG2_CHUNKS        0x00008000 ///< Input bitstream might be truncated at a packet boundaries instead of only at frame boundaries.
  #define CODEC_FLAG2_NON_LINEAR_QUANT 0x00010000 ///< Use MPEG-2 nonlinear quantizer.
  #define CODEC_FLAG2_BIT_RESERVOIR 0x00020000 ///< Use a bit reservoir when encoding if possible
+ //MEANX: NEVER EVER USE CLOSED GOP ?
+ #define CODEC_FLAG2_32_PULLDOWN   0x80000000 
  
  /* Unsupported options :
   *              Syntax Arithmetic coding (SAC)
***************
*** 1233,1238 ****
--- 1235,1241 ----
       * - decoding: unused
       */
      int rc_max_rate;
+     int rc_max_rate_header; /*< That one is set in the header MEANX */
  
      /**
       * minimum bitrate
***************
*** 1247,1252 ****
--- 1250,1257 ----
       * - decoding: unused
       */
      int rc_buffer_size;
+     int rc_buffer_size_header;  /*< That one is set in the header MEANX*/
+ 
      float rc_buffer_aggressivity;
  
      /**
***************
*** 2325,2331 ****
  
  #if LIBAVCODEC_VERSION_INT < ((52<<16)+(0<<8)+0)
  /* YUV420 format is assumed ! */
! 
  /**
   * @deprecated Use the software scaler (swscale) instead.
   */
--- 2330,2336 ----
  
  #if LIBAVCODEC_VERSION_INT < ((52<<16)+(0<<8)+0)
  /* YUV420 format is assumed ! */
! #if 0 // MEANX
  /**
   * @deprecated Use the software scaler (swscale) instead.
   */
***************
*** 2357,2363 ****
   * @deprecated Use the software scaler (swscale) instead.
   */
  attribute_deprecated void img_resample_close(struct ImgReSampleContext *s);
! 
  #endif
  
  /**
--- 2362,2368 ----
   * @deprecated Use the software scaler (swscale) instead.
   */
  attribute_deprecated void img_resample_close(struct ImgReSampleContext *s);
! #endif //MEANX
  #endif
  
  /**
