Index: libavcodec/vc1dec.c
===================================================================
--- libavcodec/vc1dec.c	(revision 23514)
+++ libavcodec/vc1dec.c	(working copy)
@@ -3118,7 +3118,10 @@
     AVFrame *pict = data;
     uint8_t *buf2 = NULL;
     const uint8_t *buf_start = buf;
-
+/** MEANX **/
+    if(s->flags& CODEC_FLAG_LOW_DELAY)
+        s->low_delay=1;
+/** /MEANX **/
     /* no supplementary picture */
     if (buf_size == 0) {
         /* special case for last picture */
