*** libavutil/internal.h.old	Sat Jan 24 22:57:51 2009
--- libavutil/internal.h	Sat Jan 24 22:57:50 2009
***************
*** 222,233 ****
--- 222,235 ----
  #endif
  
  /* avoid usage of various functions */
+ #if 0 //MEANX
  #undef  malloc
  #define malloc please_use_av_malloc
  #undef  free
  #define free please_use_av_free
  #undef  realloc
  #define realloc please_use_av_realloc
+ #endif
  #undef  time
  #define time time_is_forbidden_due_to_security_issues
  #undef  rand
