*** libavcodec/h264.c.old	Mon Feb  2 19:49:51 2009
--- libavcodec/h264.c	Mon Feb  2 19:49:50 2009
***************
*** 8010,8015 ****
--- 8010,8026 ----
      return 0;
  }
  
+ /* MEANX */
+ int av_getAVCStreamInfo(AVCodecContext *avctx, uint32_t  *nalSize, uint32_t *isAvc)
+ {
+       H264Context *h = avctx->priv_data;
+       assert(h);
+       *nalSize=h->nal_length_size;
+       *isAvc=h->is_avc;
+ 	return 1;
+ 
+   }
+ /* MeanX */
  
  AVCodec h264_decoder = {
      "h264",
