*** libavformat/movenc.c.old	Sat Jan 24 21:36:42 2009
--- libavformat/movenc.c	Sat Jan 24 21:36:41 2009
***************
*** 1757,1759 ****
--- 1757,1771 ----
      .codec_tag = (const AVCodecTag*[]){ff_mp4_obj_type, 0},
  };
  #endif
+ /* MEANX */
+ int movenc_init(void)
+ {
+     av_register_output_format(&mov_muxer);
+     av_register_output_format(&tgp_muxer);
+     av_register_output_format(&mp4_muxer);
+     av_register_output_format(&psp_muxer);
+     av_register_output_format(&tg2_muxer);
+     av_register_output_format(&ipod_muxer);
+     return 0;
+ }
+ /* MEANX */
