*** libavcodec/h264.c.old	Sun Feb 28 18:41:04 2010
--- libavcodec/h264.c	Sun Feb 28 18:41:04 2010
***************
*** 3117,3122 ****
--- 3117,3133 ----
      return 0;
  }
  
+ /* MEANX */
+ int av_getAVCStreamInfo(AVCodecContext *avctx, uint32_t  *nalSize, uint32_t *isAvc)
+ {
+       H264Context *h = avctx->priv_data;
+       assert(h);
+       *nalSize=h->nal_length_size;
+       *isAvc=h->is_avc;
+ 	return 1;
+ 
+   }
+ /* MeanX */
  
  AVCodec h264_decoder = {
      "h264",
