*** libavcodec/avcodec.h.old	Sat Jan 23 11:31:04 2010
--- libavcodec/avcodec.h	Sat Jan 23 11:31:05 2010
***************
*** 584,589 ****
--- 584,591 ----
  #define CODEC_FLAG2_NON_LINEAR_QUANT 0x00010000 ///< Use MPEG-2 nonlinear quantizer.
  #define CODEC_FLAG2_BIT_RESERVOIR 0x00020000 ///< Use a bit reservoir when encoding if possible
  #define CODEC_FLAG2_MBTREE        0x00040000 ///< Use macroblock tree ratecontrol (x264 only)
+ //MEANX: NEVER EVER USE CLOSED GOP ?
+ #define CODEC_FLAG2_32_PULLDOWN   0x80000000
  
  /* Unsupported options :
   *              Syntax Arithmetic coding (SAC)
***************
*** 1450,1455 ****
--- 1452,1458 ----
       * - decoding: unused
       */
      int rc_max_rate;
+     int rc_max_rate_header; /*< That one is set in the header MEANX */
  
      /**
       * minimum bitrate
***************
*** 1464,1469 ****
--- 1467,1474 ----
       * - decoding: unused
       */
      int rc_buffer_size;
+     int rc_buffer_size_header;  /*< That one is set in the header MEANX*/
+ 
      float rc_buffer_aggressivity;
  
      /**
