*** libavcodec/avcodec.h.old	Sat Mar 27 20:05:32 2010
--- libavcodec/avcodec.h	Sat Mar 27 20:05:32 2010
***************
*** 594,599 ****
--- 594,601 ----
  #define CODEC_FLAG2_MBTREE        0x00040000 ///< Use macroblock tree ratecontrol (x264 only)
  #define CODEC_FLAG2_PSY           0x00080000 ///< Use psycho visual optimizations.
  #define CODEC_FLAG2_SSIM          0x00100000 ///< Compute SSIM during encoding, error[] values are undefined.
+ //MEANX: NEVER EVER USE CLOSED GOP ?
+ #define CODEC_FLAG2_32_PULLDOWN   0x80000000
  
  /* Unsupported options :
   *              Syntax Arithmetic coding (SAC)
***************
*** 1464,1469 ****
--- 1466,1472 ----
       * - decoding: unused
       */
      int rc_max_rate;
+     int rc_max_rate_header; /*< That one is set in the header MEANX */
  
      /**
       * minimum bitrate
***************
*** 1478,1483 ****
--- 1481,1488 ----
       * - decoding: unused
       */
      int rc_buffer_size;
+     int rc_buffer_size_header;  /*< That one is set in the header MEANX*/
+ 
      float rc_buffer_aggressivity;
  
      /**
