--- ../ffmpeg-0.9/libavcodec/vc1dec.c	2011-12-14 15:00:29 +0000
+++ libavcodec/vc1dec.c	2011-12-14 15:00:29 +0000
@@ -5437,6 +5437,11 @@
     if(s->flags & CODEC_FLAG_LOW_DELAY)
         s->low_delay = 1;
 
+	/** MEANX **/
+	if(s->flags & CODEC_FLAG_LOW_DELAY)
+		s->low_delay = 1;
+	/** /MEANX **/
+
     /* no supplementary picture */
     if (buf_size == 0 || (buf_size == 4 && AV_RB32(buf) == VC1_CODE_ENDOFSEQ)) {
         /* special case for last picture */
