*** libavformat/Makefile.old	Sun Nov 28 09:30:28 2010
--- libavformat/Makefile	Sun Nov 28 09:30:28 2010
***************
*** 114,120 ****
                                              riff.o isom.o rmdec.o rm.o
  OBJS-$(CONFIG_MATROSKA_MUXER)            += matroskaenc.o matroska.o \
                                              riff.o isom.o avc.o \
!                                             flacenc_header.o
  OBJS-$(CONFIG_MD5_MUXER)                 += md5enc.o
  OBJS-$(CONFIG_MJPEG_DEMUXER)             += rawdec.o
  OBJS-$(CONFIG_MJPEG_MUXER)               += rawenc.o
--- 114,120 ----
                                              riff.o isom.o rmdec.o rm.o
  OBJS-$(CONFIG_MATROSKA_MUXER)            += matroskaenc.o matroska.o \
                                              riff.o isom.o avc.o \
!                                             flacenc_header.o avlanguage.o
  OBJS-$(CONFIG_MD5_MUXER)                 += md5enc.o
  OBJS-$(CONFIG_MJPEG_DEMUXER)             += rawdec.o
  OBJS-$(CONFIG_MJPEG_MUXER)               += rawenc.o
