*** libavformat/movenc.c	Sat Jan 17 21:45:18 2009
--- libavformat/movenc.c	Sat Jan 17 22:01:23 2009
***************
*** 1747,1749 ****
--- 1747,1761 ----
      .codec_tag = (const AVCodecTag*[]){ff_mp4_obj_type, 0},
  };
  #endif
+ /* MEANX */
+ int movenc_init(void)
+ {
+     av_register_output_format(&mov_muxer);
+     av_register_output_format(&tgp_muxer);
+     av_register_output_format(&mp4_muxer);
+     av_register_output_format(&psp_muxer);
+     av_register_output_format(&tg2_muxer);
+     av_register_output_format(&ipod_muxer);
+     return 0;
+ }
+ /* MEANX */
