*** libavcodec/avcodec.h.old	Sat Jan 24 21:36:29 2009
--- libavcodec/avcodec.h	Sat Jan 24 21:36:28 2009
***************
*** 460,465 ****
--- 460,467 ----
  #define CODEC_FLAG2_CHUNKS        0x00008000 ///< Input bitstream might be truncated at a packet boundaries instead of only at frame boundaries.
  #define CODEC_FLAG2_NON_LINEAR_QUANT 0x00010000 ///< Use MPEG-2 nonlinear quantizer.
  #define CODEC_FLAG2_BIT_RESERVOIR 0x00020000 ///< Use a bit reservoir when encoding if possible
+ //MEANX: NEVER EVER USE CLOSED GOP ?
+ #define CODEC_FLAG2_32_PULLDOWN   0x80000000 
  
  /* Unsupported options :
   *              Syntax Arithmetic coding (SAC)
***************
*** 1228,1233 ****
--- 1230,1236 ----
       * - decoding: unused
       */
      int rc_max_rate;
+     int rc_max_rate_header; /*< That one is set in the header MEANX */
  
      /**
       * minimum bitrate
***************
*** 1242,1247 ****
--- 1245,1252 ----
       * - decoding: unused
       */
      int rc_buffer_size;
+     int rc_buffer_size_header;  /*< That one is set in the header MEANX*/
+ 
      float rc_buffer_aggressivity;
  
      /**
***************
*** 2316,2322 ****
  
  #if LIBAVCODEC_VERSION_INT < ((52<<16)+(0<<8)+0)
  /* YUV420 format is assumed ! */
! 
  /**
   * @deprecated Use the software scaler (swscale) instead.
   */
--- 2321,2327 ----
  
  #if LIBAVCODEC_VERSION_INT < ((52<<16)+(0<<8)+0)
  /* YUV420 format is assumed ! */
! #if 0 // MEANX
  /**
   * @deprecated Use the software scaler (swscale) instead.
   */
***************
*** 2348,2354 ****
   * @deprecated Use the software scaler (swscale) instead.
   */
  attribute_deprecated void img_resample_close(struct ImgReSampleContext *s);
! 
  #endif
  
  /**
--- 2353,2359 ----
   * @deprecated Use the software scaler (swscale) instead.
   */
  attribute_deprecated void img_resample_close(struct ImgReSampleContext *s);
! #endif //MEANX
  #endif
  
  /**
