*** libavformat/movenc.c.old	Thu Jan  7 19:26:12 2010
--- libavformat/movenc.c	Thu Jan  7 19:26:12 2010
***************
*** 2138,2140 ****
--- 2138,2152 ----
      .codec_tag = (const AVCodecTag* const []){codec_ipod_tags, 0},
  };
  #endif
+ /* MEANX */
+ int movenc_init(void)
+ {
+     av_register_output_format(&mov_muxer);
+     av_register_output_format(&tgp_muxer);
+     av_register_output_format(&mp4_muxer);
+     av_register_output_format(&psp_muxer);
+     av_register_output_format(&tg2_muxer);
+     av_register_output_format(&ipod_muxer);
+     return 0;
+ }
+ /* MEANX */
