--- ../ffmpeg-0.8.4/libavcodec/h264.c	2011-09-22 20:39:58 +0100
+++ libavcodec/h264.c	2011-09-22 20:39:58 +0100
@@ -4217,6 +4217,20 @@
     { FF_PROFILE_UNKNOWN },
 };
 
+/* MEANX */
+int av_getAVCStreamInfo(AVCodecContext *avctx, uint32_t  *nalSize, uint32_t *isAvc);
+
+int av_getAVCStreamInfo(AVCodecContext *avctx, uint32_t  *nalSize, uint32_t *isAvc)
+{
+      H264Context *h = avctx->priv_data;
+      assert(h);
+      *nalSize=h->nal_length_size;
+      *isAvc=h->is_avc;
+	return 1;
+
+  }
+/* MeanX */
+
 AVCodec ff_h264_decoder = {
     "h264",
     AVMEDIA_TYPE_VIDEO,
