*** libavformat/flvenc.c.old	Sat Jan 24 22:57:46 2009
--- libavformat/flvenc.c	Sat Jan 24 22:57:45 2009
***************
*** 378,384 ****
      "video/x-flv",
      "flv",
      sizeof(FLVContext),
! #ifdef CONFIG_LIBMP3LAME
      CODEC_ID_MP3,
  #else // CONFIG_LIBMP3LAME
      CODEC_ID_ADPCM_SWF,
--- 378,385 ----
      "video/x-flv",
      "flv",
      sizeof(FLVContext),
! //MEANX #ifdef CONFIG_LIBMP3LAME
! #ifdef HAVE_LIBMP3LAME  
      CODEC_ID_MP3,
  #else // CONFIG_LIBMP3LAME
      CODEC_ID_ADPCM_SWF,
***************
*** 390,392 ****
--- 391,399 ----
      .codec_tag= (const AVCodecTag* const []){flv_video_codec_ids, flv_audio_codec_ids, 0},
      .flags= AVFMT_GLOBALHEADER,
  };
+ // MEANX
+ int flvenc_init(void)
+ {
+     av_register_output_format(&flv_muxer);
+     return 0;
+ }
\ No newline at end of file
